core/arch/arm/plat-imx/registers/imx8q.h

```diff
@@ -0,0 +1,17 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
+/*
+ * Copyright 2018-2019 NXP
+ */
+
+#ifndef __IMX8Q_H__
+#define __IMX8Q_H__
+
+#define GICD_BASE	0x51a00000
+#define GICR_BASE	0x51b00000
+#define UART0_BASE	0x5a060000
+#define UART1_BASE	0x5a070000
+#define UART2_BASE	0x5a080000
+#define UART3_BASE	0x5a090000
+#define CAAM_BASE	0x30900000
+
```

Comment:
For Aarch64 i.MX platforms, this should finally be removed I think. Let SPL pass dtb to OP-TEE, but I am fine with this change currently.